readthedocs/core/symlink.py

```diff
@@ -85,9 +85,8 @@ def __init__(self, project):
         )
         self.sanity_check()
 
-    def _log(self, msg, level='info'):
-        logger = getattr(log, level)
-        logger(constants.LOG_TEMPLATE
+    def _log(self, msg):
```

Comment:
I think the original issue means to remove this method and do each call directly to log.info.

Reply:
Hmm, apologies but I am confused about this. The issue's primary concern seems to be an external logger mangling with the built-in one. This fixes this.

Inline-ing means, it would look like:

```
log.info(constants.LOG_TEMPLATE.format(project=self.project.slug, version='',msg=msg))
```

Why not use a helper method instead?

Reply:
When we log with a helper method, all of our logging messages reference the file/lineno location of the `_log` method, not where we actually called the log method from.

Reply:
@agjohnson @stsewd Ah I see, I have made the necessary changes.